include/prlib/prlib.h

```diff
@@ -69,39 +69,119 @@ void PrCleanupModule();
 
 PR_SCENEHANDLE PrInitializeScene(sceGsDrawEnv1 *drawEnv, char *name, u_int fbp);
 void PrInitializeSceneDBuff(sceGsDBuff *buff, char *name, u_int fbp);
+
 void PrCleanupScene(PR_SCENEHANDLE scene);
 
-void PrSetSceneFrame(PR_SCENEHANDLE model, sceGsFrame frame);
-void PrSetSceneEnv(PR_SCENEHANDLE model, sceGsDrawEnv1 *drawEnv);
+void PrSetSceneFrame(PR_SCENEHANDLE scene, sceGsFrame frame);
+void PrSetSceneEnv(PR_SCENEHANDLE scene, sceGsDrawEnv1 *drawEnv);
+
 void PrPreprocessSceneModel(PR_SCENEHANDLE scene);
 
-PR_MODELHANDLE PrInitializeModel(PR_MODELHANDLE model, PR_SCENEHANDLE scene);
-PR_ANIMATIONHANDLE PrInitializeAnimation(PR_ANIMATIONHANDLE animation);
-PR_CAMERAHANDLE PrInitializeCamera(PR_CAMERAHANDLE camera);
+PR_MODELHANDLE PrInitializeModel(void* modelImage, PR_SCENEHANDLE scene);
+PR_ANIMATIONHANDLE PrInitializeAnimation(void* animImage);
+PR_CAMERAHANDLE PrInitializeCamera(void* cameraImage);
```

Comment:
Why did you remove the typedefs here?

Reply:
There's no suitable type for modelImage (PR_MODELHANDLE is reserved for PrModelObject, not SpmFileHeader) so I did that for PrInitializeModel. Definitely forgot the types for the others though lol